packages/medusa/src/interfaces/payment-service.ts

```diff
@@ -76,6 +77,9 @@ export abstract class AbstractPaymentService
   ): Promise<PaymentSessionData>
 
   public abstract createPayment(cart: Cart): Promise<PaymentSessionData>
+  public abstract createPaymentNew(
```

Comment:
todo: We've introduced a new method argument "intentRequestData" in both createPayment and createPaymentNew, and they should be reflected here in the payment service interface if I am not mistaken.

Reply:
The plan is to not change the original method `createPayment`. That one would keep using the Cart as a parameter.